Sources/WalletConnectVerify/VerifyClientFactory.swift

```diff
@@ -1,24 +1,31 @@
 import Foundation
 import WalletConnectUtils
 
-@available(iOS 14.0, *)
-@available(macOS 11.0, *)
 public class VerifyClientFactory {
-
-    public static func create() throws -> VerifyClient {
-        let originVerifier = OriginVerifier()
+    public static func create(verifyHost: String?) throws -> VerifyClient? {
+        guard let verifyHost else {
+            return nil
+        }
```

Comment:
wouldn't be better to just do like?:
```suggestion
    public static func create(verifyHost: String) throws -> VerifyClient? {
```